### PR TITLE
feat(k8s): use stable versions

### PR DESCRIPTION
**Pull request by georgettica**
now we don't need to worry about touching the action versions anymore (until it breaks something on our end

### CODE REVIEW

**Comment by georgettica** [comment]
seems they released a version that returns a 404 😱

File: .github/workflows/k8s.yaml [hunk 1]
```diff
@@ -27,9 +27,10 @@ jobs:
       - name: Checkout code
         uses: actions/checkout@v2
       - name: Start minikube
-        uses: medyagh/setup-minikube@v0.0.3
+        uses: medyagh/setup-minikube@0ff5b450588f2aee88fd0daba616d04a5582e490
```
Comment (medyagh) [inline .github/workflows/k8s.yaml:30]: this needs to be
medyagh/setup-minikube@v0.0.7
or 
medyagh/setup-minikube@latest

here is example:
https://github.com/medyagh/test-minikube-example/pull/7

https://github.com/medyagh/test-minikube-example/runs/4621762363?check_suite_focus=true

**Review by medyagh — CHANGES_REQUESTED** [review]

File: .github/workflows/k8s.yaml (hunk 1 above)
Comment (georgettica) [reply .github/workflows/k8s.yaml]: As expected, doesn't work 😢 
https://github.com/change-metrics/monocle/runs/4623008565?check_suite_focus=true

Comment (georgettica) [reply .github/workflows/k8s.yaml]: Also removing the k8s-version didn't work :(

Comment (georgettica) [reply .github/workflows/k8s.yaml]: Seems moving from stable to latest works

**Comment by georgettica** [comment]
I'll squash them Sunday.
No need to merge like this

**Comment by morucci** [comment]
Hi @georgettica thanks for the change. Would you like to squash them ?
Should we wait to be able to use stable instead of latest ?

**Comment by georgettica** [comment]
Actually, using stable is good enough.. the docs are not as clear as they should and this is the format taking the most recent version of k8s and minikube for testing

I'll squash tomorrow

**Comment by georgettica** [comment]
@morucci squashed now, let's hope it passes in a first try

**Review by morucci — APPROVED** [review]

**Pull request merged**